plugins/org.wso2.developerstudio.visualdatamapper.diagram/src/org/wso2/developerstudio/datamapper/diagram/edit/parts/TreeNode2EditPart.java

```diff
@@ -96,6 +96,81 @@
 	private static final String JSON_SCHEMA_OBJECT_VALUE_TYPE = "object_value_type";
 	private static final String JSON_SCHEMA_ARRAY_ITEMS_TYPE = "items_type";
 	private static final String NULL_VALUE = "null";
+	
+	private static final String CREATE_DATA_MAPPER_LINK1_CREATION_TOOL = "createDataMapperLink1CreationTool";
+	private static final String ELEMENT_ICON = "icons/gmf/element.png";
+	private static final String ATTRIBUTE_ICON = "icons/gmf/attribute.png";
+	private static final String ARRAY_ICON = "icons/gmf/array.png";
+	private static final String OBJECT_ICON = "icons/gmf/object.png";
+	private static final String NULLABLE_ELEMENT_ICON = "icons/gmf/element-null.png";
+	private static final String NULLABLE_ATTRIBUTE_ICON = "icons/gmf/attribute-null.png";
+	private static final String NULLABLE_ARRAY_ICON = "icons/gmf/array-null.png";
+	private static final String NULLABLE_OBJECT_ICON = "icons/gmf/object-null.png";
+	private static final String ORG_WSO2_DEVELOPERSTUDIO_VISUALDATAMAPPER_DIAGRAM = "org.wso2.developerstudio.visualdatamapper.diagram";
+	private static final String TRUE = "true";
+	private static final String FALSE = "false";
+	private static final String JSON_SCHEMA_NULLABLE = "nullable";
+
+	private static final String PREFIX = "@";
+	private static final String SCHEMA_TYPE_PREFIX = "  :[";
+	private static final String SCHEMA_TYPE_POSTFIX = "]";
+
+	static ImageDescriptor elementImgDescCollapse;
+	static ImageDescriptor attributeImgDesc;
+	static ImageDescriptor arrayImgDesc;
+	static ImageDescriptor objectImgDesc;
+	static ImageDescriptor nullableElementImgDesc;
+	static ImageDescriptor nullableAttributeImgDesc;
+	static ImageDescriptor nullableObjectImgDesc;
+	static ImageDescriptor nullableArrayImgDesc;
+	static ImageFigure attributeImg; 
+	static ImageFigure arrayImg;
+	static ImageFigure objectImg; 
+	static ImageFigure nullableObjectImg; 
+	static ImageFigure nullableArrayImg;
+	static ImageFigure nullableAttributeImg; 
+	static ImageFigure nullableElementImg;
+	static ImageFigure elementImg;
+	static {
```

Comment:
Best approach is to use org.eclipse.swt.graphics.Resource.dispose()[1] method to disposes operating system resources associated with this resource rather creating them in a static method.

Eg: attributeImg.getImage().dispose();
Please refer to https://github.com/wso2/devstudio-tooling-esb/blob/master/plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/ImportCloudConnectorWizardPage.java#L335 as well.

[1] - https://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fswt%2Fgraphics%2FResource.html